src/ui/oled/oled.c

```diff
@@ -253,7 +254,8 @@ void oled_mirror(bool mirror)
     }
 #elif PLATFORM_BITBOXBASE == 1
     (void)mirror;
-    Abort("BitBoxBase cannot mirror screen");
+    // DON'T ABORT HERE, this is called in the bootloader
+    screen_print_debug("BitBoxBase cannot mirror screen", 5000);
```

Comment:
`#else #error` also missing here :laughing: but let's merge this so we can get on with it :D

Reply:
But here it is definitely a compile error because "mirror" would be unused!